frontend/admin/src/js/components/home/HomePage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { HomeIcon, LoginIcon } from "@heroicons/react/outline";
+import CardLink from "@common/components/CardLink";
+import PageHeader from "@common/components/PageHeader";
+import { navigate } from "@common/helpers/router";
```

Comment:
I'm going to come back to this. It is unused since it commented out the line that uses `navigate` until we decide about the behaviour for a logged in user.

So, either it will be used eventually or removed.